### PR TITLE
A user can't request a friendship request from a user with a pending friendship request.

### PR DESCRIPTION
**Pull request by uri-rodberg**
- Fixed typo.
- models - a user can't request a friendship request from a user with a pending friendship request.
- Changed error messages.
- Removed method can_request_send.
- Updated tests.

https://github.com/revsys/django-friendship/issues/114

### CODE REVIEW

File: friendship/models.py [hunk 1]
```diff
@@ -337,18 +340,6 @@ def add_friend(self, from_user, to_user, message=None):
 
         return request
 
-    def can_request_send(self, from_user, to_user):
```
Comment (fcurella) [inline friendship/models.py:340]: I'm not sure I'd remove this method. One hand, it's not documented in the README, so removing it wouldn't break compatibility. OTOH, I can see its usefulness and maybe it should be documented

Comment (uri-rodberg) [reply friendship/models.py:340]: I think the name of this method implies that the user can send a request (if it returns true). But it returns the opposite - if a request was already sent. Therefore, if you decide to keep it, I would consider changing its name.

Comment (fcurella) [reply friendship/models.py:340]: +1 to changing the name, since it's undocumented anyway.

Comment (uri-rodberg) [reply friendship/models.py:340]: Also, if you keep this method, maybe it's better to check both directions of the request - from user A to B, and from B to A.

**Review by frankwiles — APPROVED** [review]

**Pull request merged**